(cmd) python debug\debugger_membp_singlestep.py
Got exception EXCEPTION_BREAKPOINT(0x80000003) at 0x7ff8e5aebd44
Got exception UNKNOW_EXCEPTION(0x4000001f) at 0x77e58727
Instruction at <0xa50006> wrote at <0xa60000>
Got single_step UNKNOW_EXCEPTION(0x4000001e) at 0xa5000c
Got single_step UNKNOW_EXCEPTION(0x4000001e) at 0xa50011
Instruction at <0xa50011> wrote at <0xa60004>
Got single_step UNKNOW_EXCEPTION(0x4000001e) at 0xa50017
Got single_step UNKNOW_EXCEPTION(0x4000001e) at 0xa5001c
Got single_step UNKNOW_EXCEPTION(0x4000001e) at 0xa50022
Got single_step UNKNOW_EXCEPTION(0x4000001e) at 0xa50023
No more single step: exiting
